django_cloud_deploy/cli/prompt.py

```diff
@@ -19,39 +19,47 @@
 import random
 import re
 import string
-from typing import Any, Dict
+import time
+from typing import Any, Dict, List
+import webbrowser
 
 from google.auth import credentials
 
 from django_cloud_deploy.cli import io
 from django_cloud_deploy.cloudlib import auth
+from django_cloud_deploy.cloudlib import billing
 
 
 class Prompt(object):
     """Base class for classes the collect user input from the console."""
 
     @classmethod
     @abc.abstractmethod
-    def prompt(cls, console: io.IO, step_prompt: str,
-               arguments: Dict[str, Any]) -> Any:
+    def prompt(
+            cls, console: io.IO, step_prompt: str, arguments: Dict[str, Any],
+            credentials: credentials.Credentials = None) -> Any:
         """Prompt the user to enter some information.
 
         Args:
             console: Object to use for user I/O.
             step_prompt: A prefix showing the current step number e.g. "[1/3]".
             arguments: The arguments that have already been collected from the
                 user e.g. {"project_id", "project-123"}
+            credentials: (Optional) The OAuth2 Credentials object to use for
```

Comment:
How about you remove (Optional) and change the type to Optional[credentials.Credentials]?